lib/src/wolt_modal_sheet.dart

```diff
@@ -213,6 +213,9 @@ class _WoltModalSheetState extends State<WoltModalSheet> {
   Widget build(BuildContext context) {
     final themeData = Theme.of(context).extension<WoltModalSheetThemeData>();
     final defaultThemeData = WoltModalSheetDefaultThemeData(context);
+    final MaterialLocalizations localizations =
+        MaterialLocalizations.of(context);
+    final String routeLabel = localizations.dialogLabel;
```

Comment:
@AcarFurkan This information should come from `WoltModalType`.

```dart
enum WoltModalType {
  bottomSheet,
  dialog;

  const WoltModalType();
  
  String routeLabel(BuildContext context) {
    final MaterialLocalizations localizations =
    MaterialLocalizations.of(context);
    switch (this) {
      case WoltModalType.bottomSheet:
        return localizations.bottomSheetLabel;
      case WoltModalType.dialog:
        return localizations.dialogLabel;
    }
  }
```
and here `_modalType(context).routeLable`

Reply:
thank you, you are right l moved the code